packages/fluentui/react-northstar/src/components/Input/Input.tsx

```diff
@@ -137,6 +155,7 @@ const Input: React.FC<WithAsProp<InputProps>> & FluentComponentStaticProps<Input
     initialValue: '',
   });
   const hasValue: boolean = !!value && (value as string)?.length !== 0;
+  const requiredAndSatisfactory = required && !!successIndicator && hasValue;
```

Comment:
```suggestion
  const requiredAndSuccess = required && !!successIndicator && hasValue;
```
Since we have `successIndicator` ?

Reply:
Done